PCbuild/fix_lineendings.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python3
+#
+# Fixes line endings of batch files to ensure they remain LF.
```

Comment:
This isn't right, they should be CRLF :).  Also, a `del .git\index && git reset` from a clean (no changes, no untracked or ignored files) checkout should give you proper line endings according to the rules in `.gitattributes`.